indico/modules/rb/client/js/common/rooms/edit/RoomEditModal.jsx

```diff
@@ -102,6 +104,17 @@ function RoomEditModal({roomId, locationId, onClose, afterCreation}) {
     );
   }, [roomId]);
 
+  useEffect(() => {
```

Comment:
I'd include it in this block here to avoid toggling the loading state multiple times

```js
  useEffect(() => {
    (async () => {
      if (!isNewRoom) {
        setLoading(true);
        await fetchRoomData();
        setLoading(false);
      }
    })();
  }, [isNewRoom, fetchRoomData]);
```